### ISSUE TITLE
[Question] Integration with immersive portals

### ISSUE THREAD

**Issue by Alomare** [Palyon-dev/CustomPortals#16]
I didn't know where to leave such a simple comment, but the GitHub issues were the only option. I'm playing with this mod and Immersive Portals at the same time, and I'm aweare of the cross-dimensions portal bug. However, I was wondering if there will be some sort of integration in the future where custom portals work like imm. portals, with all the rendering tricks. It would be awesome.

Its apparently quite simple already to create custom portals with imm. portals, but its creative-only and doesn't have the charm of this mod.

I know little about mod dev but I assume that custom portals innerworkings are much simpler, using only portal animations and teleporting (correct me if I'm wrong!) so I wouldn't expect this to happen. Still curious :)

Love your mod

**Comment by Palyon-dev** [comment]
Thanks! I think I tried integrating my mod with Immersive Portals a long time ago and couldn't quite figure it out. Granted, I didn't try _super_ hard, but it's never been a major priority for me with this project. Additionally, it presents some design issues: It would kind of make the whole aspect of portals coming in all sorts of different colors not make sense anymore, since there would be no portal blocks and you'd just look right through the portals. It would also make haste runes essentially a dead item. I'm also not sure if Immersive Portals supports portals in shapes other than rectangles like my mod does, so that could present another issue.

Long story short, I don't have any plans to implement this unfortunately.
